sdk/src/main/java/com/mapbox/maps/MapProjectionUtils.kt

```diff
@@ -18,7 +17,7 @@ internal object MapProjectionUtils {
     return when (map?.get(NAME_KEY)?.contents as? String) {
       MERCATOR_PROJECTION_NAME -> MapProjection.Mercator
       GLOBE_PROJECTION_NAME -> MapProjection.Globe
-      else -> throw RuntimeException("Could not cast given Value to valid MapProjection!")
+      else -> throw MapboxMapException("Could not cast given Value to valid MapProjection!")
```

Comment:
Does ClassCastException fit here?

Reply:
I was thinking about it and imho answer is no - we don't cast classes directly. Let's leave this exception type